core/utils.c

```diff
@@ -218,7 +218,7 @@ size_t utils_intToText(int64_t data,
             string[int64_min_strlen] = '\0';
             return int64_min_strlen;
         }
-        result = utils_uintToText((uint64_t)labs(data), string + 1, length - 1);
+        result = utils_uintToText((uint64_t)llabs(data), string + 1, length - 1);
```

Comment:
I think the casts to uint64_t are pointless here (and in some other places in the code base). Best case they do nothing, worst case they silence some helpful warnings. How about removing them?